tests/cases/whitespace/method-call

```diff
@@ -3,25 +3,36 @@ Method calls must not have unnecessary whitespace
 ---CONTENTS---
 <?php
 
-$foo -> bar ( $arg1 );
+$foo -> bar ( $arg1 ) -> baz ( );
 
 $foo
 -> bar ($arg1
 ,$arg2)
 -> baz ($arg1
 ,$arg2);
 
+$fooBar()
+    ->baz()
+    ->qux()
+;
+
 ---FIXED---
 <?php
 
-$foo->bar($arg1);
+$foo->bar($arg1)->baz();
```

Comment:
Would rather these became multiline.

Reply:
It is reasonable for me to allow an existing single-line statement to remain single-line, while if you want to go multi-line then you would have to follow a standard.